GovUk.Frontend.Umbraco.ExampleApp/Views/Shared/_Layout.cshtml

```diff
@@ -3,10 +3,12 @@
 @using Microsoft.AspNetCore.Localization
 @using Microsoft.Extensions.Configuration;
 @using ThePensionsRegulator.Frontend.Umbraco.Models;
+@using ThePensionsRegulator.Frontend.Umbraco.Services
 @using Umbraco.Cms.Core.Models
 @using Umbraco.Cms.Web.Common
 @inject UmbracoHelper Umbraco
 @inject IConfiguration Configuration
+@inject ITprGlobalNavigationSerivce NavigationSerivce;
```

Comment:
Typo "Service"

(As a variable this should really be camel case `navigationService`, although I can see you've just copied the examples above...)